presto-main/src/main/java/io/prestosql/operator/scalar/StringFunctions.java

```diff
@@ -899,4 +899,18 @@ public static Slice concat(@LiteralParameter("x") Long x, @SqlType("char(x)") Sl
 
         return result;
     }
+
+    @Description("Determine whether source starts with prefix or not")
+    @ScalarFunction("starts_with")
+    @LiteralParameters({"x", "y"})
+    @SqlType(StandardTypes.BOOLEAN)
+    public static boolean startsWith(@SqlType("varchar(x)") Slice source, @SqlType("varchar(y)") Slice prefix)
+    {
+        if (source.length() < prefix.length()) {
+            return false;
+        }
+        else {
+            return source.compareTo(0, prefix.length(), prefix, 0, prefix.length()) == 0;
+        }
```

Comment:
`else` is unnecessary:

```suggestion
        if (source.length() < prefix.length()) {
            return false;
        }
        return source.compareTo(0, prefix.length(), prefix, 0, prefix.length()) == 0;
```

Reply:
Fixed.